.gitignore

```diff
@@ -63,4 +63,4 @@ public/wallet
 # Synpress directories and files
 screenshots
 videos
-cypress
\ No newline at end of file
+videos-ci
```

Comment:
why have a second videos output folder?

Reply:
@turadg This folder is used to contain CI video recordings. I am working on the CI setup(In Progress) over here:
https://github.com/rabi-siddique/rs-wallet-app/pull/2

The video service defined in `docker-compose.yml` records activities inside the Docker container, and this folder serves as the storage location for those recordings.

Reply:
@turadg Makes sense. I'll update the `docker-compose.yml` to update the directory. Given that the file names are different for both recordings, it won't be an issue to have them inside the same folder.